src/exchange_adapters/whitebit.ts

```diff
@@ -5,7 +5,9 @@ import { Exchange } from '../utils'
 export class WhitebitAdapter extends BaseExchangeAdapter {
   baseApiUrl = 'https://whitebit.com/api/v4/public/'
   readonly _exchangeName = Exchange.WHITEBIT
-  readonly _certFingerprint256 = ''
+  // Cloudflare Inc ECC CA-3
+  readonly _certFingerprint256 =
+    '3A:BB:E6:3D:AF:75:6C:50:16:B6:B8:5F:52:01:5F:D8:E8:AC:BE:27:7C:50:87:B1:27:A6:05:63:A8:41:ED:8A'
```

Comment:
@ninocomputer did you choose the second out of three leaves in the certificate tree by choice? 

I thought the 'safest' means the lowest leaf, e.g. bitget.com certificate

<img width="239" alt="Screenshot 2023-02-27 at 12 58 19" src="https://user-images.githubusercontent.com/28403703/221558140-7508c8dc-9c24-46b3-b717-9c8d2cad5ff6.png">

Reply:
It's always the second across other adapters, so I'm just following the same pattern.